docs/src/development/environment-setup.md

```diff
@@ -27,50 +29,70 @@ Please make sure you're using Scarb installed via [asdf](https://asdf-vm.com/) -
 > ```shell
 > $HOME/.asdf/shims/scarb
 > ```
-> 
-> If you previously installed scarb using official installer, you may need to remove this installation or modify your PATH to make sure asdf installed one is always used.
-
 
-> ❗️ **Warning**
-> 
-> If you haven't pushed your branch to the remote yet (you've been working only locally), two tests will fail:
-> 
-> - `e2e::running::init_new_project_test`
-> - `e2e::running::simple_package_with_git_dependency`
+> 💡 **Info**
 > 
-> After pushing the branch to the remote, those tests should pass.
+> If you previously installed Scarb using official installer, you may need to remove that installation or modify your `PATH`
+> to make sure the version installed by asdf is always used.
+
 
 ### Starknet Devnet
-To install it run `./scripts/install_devnet.sh`
+Install it by running `./scripts/install_devnet.sh`
+
 
 ### Universal sierra compiler
 Install the latest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler) version.
 
 
+### Environmental variables
+
+Set `NODE_URL` environmental variable to a Sepolia testnet node URL:
+- either manually in your shell
+    ```shell
+    $ export NODE_URL="https://example.com" 
+    ```
+- or inside [`.cargo/config.toml` file](https://doc.rust-lang.org/cargo/reference/config.html#configuration-format)
```

Comment:
This is an option, but we must make sure our url is not commited at any point to the repo. Maybe we could suggest using dotenv and .env file instead? we alredy do so in sncast cc @ddoktorski

Reply:
also, we use the same variable but in different format in sncast (also set in .env or just in env). if we are putting this info in docs, maybe we should include the same info on sncast as well?

Reply:
Yup, changing that to `.env` file. 
What do you exactly mean by including the same info on sncast?

Reply:
We also use .env and depend on node url declared in it, but up until now we only had this info in readme. Since we are expanding the docs to include this info we should include an info it is needed for snforge and sncast (there will probably be two separate files?)

Reply:
Changed env to be `SEPOLIA_RPC_URL` to match `sncast` env file format